gadgets/traceloop/README.mdx

```diff
@@ -8,38 +8,79 @@ import TabItem from '@theme/TabItem';
 
 # traceloop
 
-The traceloop gadget is a syscalls flight recorder.
+The trace_loop gadget is used to capture and replay system calls in real-time, acting as a flight recorder for your Kubernetes applications.
 
 ## Requirements
 
-- Minimum Kernel Version : 5.10
+- Minimum Kernel Version: 5.10+
+- Dependencies: Inspektor Gadget installed
+- Access: kubectl access to your cluster
 
-## Getting started
+## Getting Started
 
 Running the gadget:
 
 <Tabs groupId="env">
     <TabItem value="kubectl-gadget" label="kubectl gadget">
-        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/traceloop:%IG_TAG% [flags]
+        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/traceloop:latest [flags]
     </TabItem>
 
     <TabItem value="ig" label="ig">
         ```bash
-        $ sudo ig run ghcr.io/inspektor-gadget/gadget/traceloop:%IG_TAG% [flags]
+        $ sudo ig run ghcr.io/inspektor-gadget/gadget/traceloop:latest [flags]
         ```
     </TabItem>
 </Tabs>
 
 ## Guide
 
-First, we need to run an application that generates some events.
+System calls are the foundation of how applications interact with the operating system. When debugging production issues, understanding the exact sequence of system calls before a crash or failure can be invaluable. The trace_loop gadget continuously captures these system calls, allowing you to "rewind" and see exactly what your application was doing when something went wrong.
+
+:::info
+Think of trace_loop as a flight recorder for your applications - it's always recording, and when something goes wrong, you can replay the exact sequence of events that led to the issue.
+:::
+
+### Flow of System Call Recording in Kubernetes
+
+Before diving into scenarios, it's important to understand how trace_loop captures system calls across different components:
+
+**Application Pods**: The primary targets that generate system calls through normal operation.
+
+**eBPF Probes**: Kernel-level probes that capture system calls in real-time.
+
+**Traceloop Recorder**: The component that buffers and formats system call data.
+
+**Output Stream**: Real-time display of captured system calls with context.
+
+The overall flow of system call capture is as follows:
+```
+Application Pods → System Calls → eBPF Probes → Traceloop Recorder → Output Stream
+     ↓                ↓              ↓               ↓              ↓
+Generate syscalls → Kernel level → Capture → Buffer & Format → Display
+```
```

Comment:
It is also possible to use mermaid diagrams. You can experiment with https://mermaid.live/ and then insert it here.

Example of gadgets using mermaid diagrams for documentation:
https://github.com/inspektor-gadget/inspektor-gadget/blob/main/gadgets/trace_tcp/README.mdx#connect